Runtime/PostProcessing/Shaders/CRT.shader

```diff
@@ -139,6 +150,91 @@ Shader "Hidden/HauntedPS1/CRT"
         return color;
     }
 
+    float3 QuadratureAmplitudeModulation(float3 colorYIQ, float2 positionFramebufferNDC)
+    {
+        float Y = colorYIQ.r;
+        float I = colorYIQ.g;
+        float Q = colorYIQ.b;
+
+        float lineNumber = 1;
+        float carrier_phase =
+            _NTSCHorizontalCarrierFrequency * positionFramebufferNDC.x +
+            _NTSCLinePhaseShift * lineNumber;
+        float s = sin(carrier_phase);
+        float c = cos(carrier_phase);
+
+        float modulated = I * s + Q * c;
+        float3 premultiplied = float3(Y, 2 * s * modulated, 2 * c * modulated);
+        
+        return premultiplied;
+    }
+
+    float Gaussian(int positionX, int halfRange)
+    {
+        return exp2(-.5 * ((float)positionX / halfRange * (float)positionX / halfRange));
+    }
+
+    float3 ComputeGaussianInYIQ(float2 positionFramebufferNDC)
+    {
+        float3 colorTotal = 0;
+        float weightTotal = 0.0;
+        
+        float2 positionFramebufferCenterPixels = positionFramebufferNDC * _ScreenSizeRasterizationRTScaled.xy;
+        int halfRange = 3;
+        for (int x = -halfRange; x <= halfRange; ++x)
+        {
+            // Convert from framebuffer normalized position to pixel position
+            float2 positionFramebufferCurrentPixels = positionFramebufferCenterPixels + float2(x * (_NTSCKernelWidthRatio * _NTSCHorizontalCarrierFrequency), 0);
+            float2 positionFramebufferCurrentNDC = positionFramebufferCurrentPixels * _ScreenSizeRasterizationRTScaled.zw;
+            
+            positionFramebufferCurrentNDC = ClampRasterizationRTUV(positionFramebufferCurrentNDC);
+            float3 colorCurrent = FCCYIQFromSRGB(FetchFrameBuffer(positionFramebufferCurrentNDC).rgb);
+
+            // Use the original offset value in order to keep it the same distance across resolutions
+            colorCurrent = QuadratureAmplitudeModulation(colorCurrent, positionFramebufferCurrentPixels);
+
+            // Fetch the Gauss weight at the current x offset position
```

Comment:
This comment doesn't actually add value, remove it. The code below describes exactly this.